test/PizzeriaTest.cpp

```diff
@@ -65,3 +75,29 @@ TEST_F(PizzeriaTest, calculatePriceForPizzaMock)
 
     delete mock;
 }
+
+TEST_F(PizzeriaTest, orderStubAndTwoMockPizzas) {
+    // Given
+    StrictMock<PizzaMock>* strict_mock = new StrictMock<PizzaMock>{"strict", 1, minutes(1)};
+    NiceMock<PizzaMock>* nice_mock = new NiceMock<PizzaMock>{"nice", 2, minutes(3)};
+    PizzaStub ps("STUB");
+    Pizzas pizzas = {&ps, strict_mock, nice_mock};
+    EXPECT_CALL(*strict_mock, getPrice()).WillRepeatedly(Return(strictMockPizzaPrice));
+    EXPECT_CALL(*strict_mock, getName());
+    EXPECT_CALL(*strict_mock, getBakingTime()).WillOnce(Return(minutes(strictMockPizzaBakingTime)));
+    EXPECT_CALL(*nice_mock, getPrice()).WillOnce(Return(niceMockPizzaPrice));
```

Comment:
you are missing expect call on nice mock - this one about baking time. This time is expected later by sleep_for below.